src/library/Uno.Themes.WinUI.Markup/Themes/Slider.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Slider
+		{
+			public static class Resources
+			{
+				public static class Default
+				{
+					public static class Typography
+					{
+						public static class FontFamily
+						{
+							[ResourceKeyDefinition(typeof(Microsoft.UI.Xaml.Media.FontFamily), "SliderFontFamily")]
+							public static ResourceValue<Microsoft.UI.Xaml.Media.FontFamily> Default => new("SliderFontFamily", true);
+						}
+
+						public static class FontSize
+						{
+							[ResourceKeyDefinition(typeof(double), "SliderFontSize")]
+							public static ResourceValue<double> Default => new("SliderFontSize", true);
+						}
+					}
+
+					public static class TrackFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFill")]
+						public static ResourceValue<Brush> Default => new("SliderTrackFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderTrackFillDisabled", true);
+					}
+
+					public static class TrackValueFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackValueFill")]
+						public static ResourceValue<Brush> Default => new("SliderTrackValueFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderTrackValueFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderTrackValueFillDisabled", true);
+					}
+
+					public static class TickBarFill
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderTickBarFill")]
+						public static ResourceValue<Brush> Default => new("SliderTickBarFill", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "SliderTickBarFillDisabled")]
+						public static ResourceValue<Brush> Disabled => new("SliderTickBarFillDisabled", true);
+					}
+
+					public static class TrackCornerRadius
+					{
+						[ResourceKeyDefinition(typeof(CornerRadius), "SliderTrackCornerRadius")]
+						public static ResourceValue<CornerRadius> Default => new("SliderTrackCornerRadius", true);
+					}
+
+					public static class InlineTickBar
+					{
+						[ResourceKeyDefinition(typeof(Brush), "SliderInlineTickBarFill")]
+						public static ResourceValue<Brush> Default => new("SliderInlineTickBarFill", true);
+					}
```

Comment:
Following the updated specs [here](https://github.com/unoplatform/uno.themes-private/issues/16#issue-1909235521), we should create a `Track` nested class that contains `Fill`, `ValueFill`, `Thickness`, `CornerRadius`, etc.

Where `Fill` and `ValueFill` are both classes that have a `Default` and a `Disabled`

And a `TickBar` nested class with things like `InlineFill`, `Fill.Default` `Fill.Disabled`, etc. placed inside of it

Reply:
Done